sdk/python/tests/test_kfp_samples.sh

```diff
@@ -73,11 +73,15 @@ rm -f "${COMPILER_OUTPUTS_FILE}"
 # compile each of the Python scripts in the KFP testdata folder
 for f in "${KFP_TESTDATA_DIR}"/*.py; do
   echo -e "\nCompiling ${f##*/}:" >> "${COMPILER_OUTPUTS_FILE}"
-  if dsl-compile-tekton --py "${f}" --output "${TEKTON_COMPILED_YAML_DIR}/${f##*/}.yaml" >> "${COMPILER_OUTPUTS_FILE}" 2>&1;
-  then
-    echo "SUCCESS: ${f##*/}" | tee -a "${COMPILER_OUTPUTS_FILE}"
+  if [ ${f##*/} == "compose.py" ] || [ ${f##*/} == "basic_no_decorator.py" ]; then
```

Comment:
Can we create a config file for all the special pipelines along with any extra information to compile it?